content/en/docs/concepts/deployment-models/index.md

```diff
@@ -0,0 +1,421 @@
+- Various [control plane models](#control-plane-models): Support different
+  levels of availability.
+- Team or project isolation: Each team runs its own set of clusters.
+
+{{< image width="50%"
+    link="./multi-cluster.svg"
+    alt="A service mesh with multiple clusters"
+    title="Multi-cluster"
+    caption="A service mesh with multiple clusters"
+    >}}
+
+Multi-cluster deployments give you a greater degree of isolation and
+availability but increase complexity. If your systems have high availability
+requirements, you likely need clusters across multiple zones and regions. You
+can canary configuration changes or new binary releases in a single cluster,
+where the configuration changes only affect a small amount of user traffic.
+Additionally, if a cluster has a problem, you can temporarily route traffic to
+nearby clusters until you address the issue.
+
+You can configure inter-cluster communication based on the
+[network](#network-models) and the options supported by your cloud provider. For
+example, if two clusters reside on the same underlying network, you can enable
+cross-cluster communication by simply configuring firewall rules.
+
+## Network models
+
+Many production systems require multiple networks or subnets for isolation
+and high availability. Istio supports spanning a service mesh over a variety of
+network topologies.
+
+### Single network
+
+In the simplest case, a service mesh operates over a single fully
+connected network. A single network model satisfies the following conditions:
+
+- The network consists of a number of connected subnets without any
+  overlap in IP or VIP ranges.
+- All {{< gloss >}}workload instance{{< /gloss >}}s can reach each other directly
+  without an Istio gateway.
+
+A single network allows Istio to configure service consumers in a uniform
+way across the mesh with the ability to directly address workload instances.
+
+{{< image width="50%"
+    link="./single-net.svg"
+    alt="A service mesh with a single network"
+    title="Single network"
+    caption="A service mesh with a single network"
+    >}}
+
+### Multiple networks
+
+You can span a single service mesh across multiple networks; such a
+configuration is known as **multi-network**.
+
+Multiple networks afford the following capabilities beyond that of single networks:
+
+- An existing network topology
+- Overlapping IP or VIP ranges for **service endpoints**
+- Crossing of administrative boundaries
+- Fault tolerance
+- Scaling of network addresses
+
+In this model, the workload instances in different networks can only reach each
+other through one or more [Istio gateways](/docs/concepts/traffic-management/#gateways).
+Istio uses **partitioned service discovery** to provide consumers a different
+view of {{< gloss >}}service endpoint{{< /gloss >}}s. The view depends on the
+network of the consumers.
+
+{{< image width="50%"
+    link="./multi-net.svg"
```

Comment:
Should we have a gateway on each side rather than one in the middle?

Reply:
I tried to keep the diagrams as simple as possible. For that reason, I place the gateway in the middle.

Reply:
I'm worried it may be confusing, since there isn't actually a shared gateway. Each network needs its own gateway.

Reply:
I agree with @smawson, gateways are not bi-directional so the diagram is confusing.

Btw, the fonts in all the diagrams are too small in the preview. I also see that the fonts in the traffic management concepts diagrams are also still not right. What's the plan for fixing them?

Reply:
@smawson @frankbu I'm changing the diagram based on your comments. Regarding the fonts in diagrams, that is an issue that is caused by the available fonts in your local system. I can convert the text to outlines but then it stops being editable and accessible to screen readers, which is terrible for accessibility reasons. How would you recommend I proceed?

Reply:
@rcaballeromx, are you saying the small fonts are only a problem for a small number of users, which I'm one of? For me, much of the text in diagrams is not really legible, without clicking to enlarge the diagram. If that's not the case for most users, then I think its OK, but if it's not just me, it seems like it needs to be fixed.

Reply:
If the concern is the size of the text, I can always enlarge the diagrams. I can also enlarge the text but that could make the diagrams appear too loaded. @frankbu I suggest I make some adjustments to the diagrams for the next update based on your feedback. We can revisit after that, would that work?